### PR TITLE
Datamodel Implementation

### PR DESCRIPTION
**Pull request by kanafghan**

### CODE REVIEW

**Comment by kanafghan** [comment]
@toin0u: You should aslo review this together with @khannoori.

File: README.md [hunk 1]
```diff
@@ -2,5 +2,30 @@
 
 The Google App Engine application used as the Backend for the Fiziq app.
 
+## Getting Started
+In order to get started, you must have the following installed on your system:
+
+- [Python 2.7.x](https://www.python.org/downloads/)
+- [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python)
+
+Start the [development web server]() as follows:
```
Comment (toin0u) [inline README.md]: I think the link is missing here isn't it ?

File: README.md [hunk 2]
```diff
@@ -2,5 +2,30 @@
...
+
+````bash
```
Comment (toin0u) [inline README.md]: only 3 backticks :) see: https://help.github.com/articles/markdown-basics/#multiple-lines

File: README.md (hunk 1 above)
Comment (kanafghan) [inline README.md]: You are right, fixed it.

File: README.md (hunk 2 above)
Comment (kanafghan) [inline README.md]: Right. Fixed.

**Comment by kanafghan** [comment]
@toin0u and @khannoori : This is now longer WIP you can merge it if possible.

**Comment by khannoori** [comment]
@toin0u i have reviewed, you can merge after you have reviewed.

File: README.md [hunk 3]
```diff
@@ -2,5 +2,107 @@
...
+## Getting Started
+In order to get started, you must have the following installed on your system:
+
+- [Python 2.7.x](https://www.python.org/downloads/)
+- [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python)
+
+Start the [development web server](https://cloud.google.com/appengine/docs/python/tools/devserver) as follows:
+
+```bash
+$ ./dev_appserver.py /path/to/fiziq-backend/src
```
Comment (toin0u) [inline README.md]: I think we can remove `./` here

File: src/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    """
+    repetitions = ndb.IntegerProperty()
+    weight = ndb.FloatProperty()
+    workout_session = ndb.KeyProperty(kind='WorkoutSession')
+    workout = ndb.KeyProperty(kind='Workout')
+
+
+class Workout(ndb.Model):
+    """
+    Representation of a physicual exercise targetting one or
```
Comment (toin0u) [inline src/models/__init__.py]: typo in physical :)

File: src/app.yaml [hunk 5]
```diff
@@ -8,10 +8,13 @@ handlers:
...
+- url: /tests.*
+  script: lib.gaeunit.gaeunit.tester
 - url: .*
   script: main.app
 
 libraries:
 - name: webapp2
   version: "2.5.2"
+- name: django
+  version: "1.4"
```
Comment (toin0u) [inline src/app.yaml]: I think a newline is missing here :)

Comment (kanafghan) [inline src/app.yaml]: Fixed.

File: src/models/__init__.py (hunk 4 above)
Comment (kanafghan) [inline src/models/__init__.py]: Fixed.

File: README.md (hunk 3 above)
Comment (kanafghan) [inline README.md]: Fixed.

**Comment by toin0u** [comment]
:+1:

**Pull request merged**